core/shared/src/main/scala/chartreuse/Plot.scala

```diff
@@ -20,20 +20,24 @@ import chartreuse.layout.ScatterPlot
 import doodle.algebra.*
 import doodle.core.*
 import doodle.syntax.all.*
+import chartreuse.Coordinate._
```

Comment:
```suggestion
import chartreuse.Coordinate.*
```

`_` is Scala 2 syntax and will be removed in the future.